music21/musicxml/m21ToXml.py

```diff
@@ -5523,7 +5522,17 @@ def articulationToXmlTechnical(self, articulationMark):
             mxTechnicalMark.text = str(articulationMark.number)
         if musicXMLTechnicalName == 'fret':
             mxTechnicalMark.text = str(articulationMark.number)
-
+        if musicXMLTechnicalName == 'bend':
+            bendAlterSubElement = SubElement(mxTechnicalMark, 'bend-alter')
+            bendAlterSubElement.text = str(articulationMark.bendAlter.semitones)
+            if articulationMark.preBend:
+                preBendSubElement = SubElement(mxTechnicalMark, 'pre-bend')
+            if articulationMark.release is not None:
+                releaseSubElement = SubElement(mxTechnicalMark, 'release')
+                releaseSubElement.set('offset', str(articulationMark.release))
```

Comment:
musicxml's definition of offset is very different from music21's and (despite me now being editor of the musicxml spec) we shouldn't use musicxml offsets anywhere in music21 -- these need to be converted according to the current divisionsPerQuarter setting.

Reply:
should work with https://github.com/cuthbertLab/music21/pull/1580/commits/d93af6d98c7b9bd1a7587c11d1c2d20f961a4460 though I need to write the tests